custom_components/ramses_cc/coordinator.py

```diff
@@ -294,9 +294,5 @@ async def async_update(self, *args, **kwargs) -> None:
         if new_sensors or new_heat_entities or new_hvac_entities:
             self.hass.helpers.event.async_call_later(5, self.async_save_client_state)
 
-        self.async_dispatcher_send()
-
-    def async_dispatcher_send(self, *args, **kwargs) -> None:
-        """Inform the devices that their state data may have changed."""
-
-        async_dispatcher_send(self.hass, DOMAIN, *args, **kwargs)
+        # Trigger state updates of all entities
+        async_dispatcher_send(self.hass, SIGNAL_UPDATE)
```

Comment:
I can't recall if the new entities do an update after instantiation - IIRC this is configurable.

Other than that, when do we need to do this? E.g. look at name of zones - it just pulls the attr from the underlying **ramses_rf** entity when required.

Reply:
This will trigger the calls to `async_write_ha_state` which I understood to be necessary as the entities are not set up to be polled (ie. `_attr_should_poll` is false) so they must push state to HA. Please correct me if I'm wrong!

Reply:
OMG - I think I was/am polling, IIRC (oeriodic call of aync_update is scheduled). But it could move to fully event-driven with callbacks from **ramses_rf** - that wont happen soon.

Reply:
Entity polling is [currently disabled in master](https://github.com/zxdavb/ramses_cc/blob/7fcf48ae23a7c5073addbfd25fd606c061ef3457/custom_components/ramses_cc/__init__.py#L190) so they will only save out their state when the broker `async_update` is called on the `CONF_SCAN_INTERVAL` schedule by way of listening for the `DOMAIN` message to be dispatched.

I haven't intentionally changed the fundamentals of this setup in this PR so if there's anything here that is not maintaining the same point of saving state as at present it was not deliberate.

I had the same thought about events from ramses_rf but also figured it wouldn't be coming soon.